### ISSUE TITLE
Samourai wallet connected to Dojo on Mynode not showing stransactions

### ISSUE THREAD

**Issue by salenjak** [Samourai-Wallet/samourai-wallet-android#399]
Hi,

I have connected Samourai wallet to Dojo on Mynode.

Everything seems to be working (All lights are green in Networking->Connections, and Dojo is enabled and running on Mynode) but Samourai wallet on my Android phone won't update transaction that I sent a few hours ago (I have exported Samourai's zpub to Electrum and it shows transaction).

Any ideas?

**Comment by salenjak** [comment]
I have just find out that Dojo V3 Onion Address listed in Mynode isn't reachable when trying to open it in Tor browser. Maybe this is some error on Mynode side?

**Comment by salenjak** [comment]
Just try to restore the wallet with seed (connected to Samourai server not my dojo) and it still can't find transactions. Performed restore with same seed on Electrum android app that is connected to my electrum server and everything works normally.

**Comment by salenjak** [comment]
Yesterday, I decided to switch off my phone for a few hours to see will it change anything. 
This morning after 8 hours I started the phone and then Samourai wallet, and everything works. Transactions and balance are here. WTF

Steps to follow if your Samourai doesn't want to show balance but shows that it is connected to Dojo end everything works OK:
Disable Electrum server
Disable Dojo
Reinstall Dojo (this worked for others with the same problem)
Leave f....g  phone switched off for few hours.
I don't know what caused my problem, but maybe some is cache, temp file or whatever.
